packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5788,6 +5802,7 @@ export default class Meeting extends StatelessWebexPlugin {
           })
           .then(() => {
             this.isSharing = true;
```

Comment:
Please remove the `isSharing` flag, it's redundant now. Also there are 2 places where `isSharing` is used and `screenShareFloorState` should be used instead now:
1. in clearMeetingData()
2. in moveTo() in the listener for LOCUSINFO.EVENTS.SELF_OBSERVING

Reply:
removed `isSharing` and added `screenShareFloorState` where it was being used